conf/app.routes

```diff
@@ -44,7 +44,7 @@ GET         /submissions/:id/view                          controllers.Submissio
 
 GET         /submissions/:id                               controllers.SubmissionsController.amend(id: String, isAmendment: Boolean)
 
-GET         /submissions/:id/:url/:pattern/:messageKey     controllers.SubmissionsController.amendErrors(id: String, url: String, pattern: String, messageKey: String, isAmendment: Boolean)
+GET         /submissions/:id/:pattern/:messageKey          controllers.SubmissionsController.amendErrors(id: String, pattern: String, messageKey: String, isAmendment: Boolean, url: RedirectUrl)
```

Comment:
What happened to the `url` path param here, where is it getting defined? Won't this break our error change links?

Reply:
Ok, I see now that it is a query string param rather than a path param :+1: